src/test/java/org/broadinstitute/hellbender/utils/sam/ArtificialSAMUtils.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.hellbender.utils.sam;
```

Comment:
Please delete this class -- it has been replaced in hellbender with `ArtificialReadUtils`